node/task.ts

```diff
@@ -407,7 +407,7 @@ export interface EndpointAuthorization {
 
 /**
  * Gets the authorization details for a service endpoint
- * If the authorization was not set and is not optional, it will throw.
+ * If the authorization was not set and is not optional, it will write logs and return undefined.
```

Comment:
Could you please elaborate more? Will in change task result also? I don't think we need to mention undefined result